packages/modal/src/components/drawer/Drawer.module.css

```diff
@@ -3,7 +3,7 @@
     --background-opacity: 0;
     position: fixed;
     inset: 0;
-    background-color: rgba(255, 255, 255, var(--background-opacity));
+    background-color: transparent;
```

Comment:
If this is what we want, we should also remove the `--background-opacity` variable.

And in that case, also remove `&.afterOpen {`, `&.beforeClose {` selectors.

Reply:
Actually, we can remove the `background-color` property altogether.

Reply:
Alright, resolved!